Composer/packages/extensions/obiformeditor/src/Form/fields/PromptField/ChoiceInput/Choices.tsx

```diff
@@ -3,239 +3,74 @@
-}
+import { StaticChoices } from './StaticChoices';
+import { DynamicChoices } from './DynamicChoices';
 
 interface ChoicesProps {
   id: string;
   schema: JSONSchema6;
-  formData?: IChoice[];
+  formData?: IChoice[] | string;
   label: string;
-  onChange: (data: IChoice[]) => void;
+  onChange: (data: IChoice[] | string) => void;
 }
 
-const ChoiceItem: React.FC<ChoiceItemProps> = props => {
-  const { choice, index, onEdit, hasMoveUp, hasMoveDown, onReorder, onDelete } = props;
-  const [key, setKey] = useState(`${choice.value}:${choice.synonyms ? choice.synonyms.join() : ''}`);
-
-  const contextItems: IContextualMenuItem[] = [
-    {
-      key: 'moveUp',
-      text: 'Move Up',
-      iconProps: { iconName: 'CaretSolidUp' },
-      disabled: !hasMoveUp,
-      onClick: () => onReorder(index, index - 1),
-    },
-    {
-      key: 'moveDown',
-      text: 'Move Down',
-      iconProps: { iconName: 'CaretSolidDown' },
-      disabled: !hasMoveDown,
-      onClick: () => onReorder(index, index + 1),
-    },
-    {
-      key: 'remove',
-      text: 'Remove',
-      iconProps: { iconName: 'Cancel' },
-      onClick: () => onDelete(index),
+export const Choices: React.FC<ChoicesProps> = props => {
+  const {
+    id,
+    label,
+    formData,
+    onChange,
+    schema: { oneOf = [] },
+  } = props;
+  const [dynamicSchema] = oneOf;
+
+  const options = useMemo(() => oneOf.map(({ title = '' }: any) => ({ key: title.toLowerCase(), text: title })), [
+    oneOf,
+  ]);
+  const [choiceType, setChoiceType] = useState(Array.isArray(formData || []) ? 'static' : 'dynamic');
+  const handleChange = useCallback(
+    (_, { key }) => {
+      onChange(choiceType !== 'static' ? [] : '');
+      setChoiceType(key);
     },
-  ];
-
-  const handleEdit = (field: 'value' | 'synonyms') => (_e: any, val?: string) => {
-    if (field === 'synonyms') {
-      onEdit(index, { ...choice, synonyms: val ? val.split(', ') : [] });
-    } else {
-      onEdit(index, { ...choice, value: val });
-    }
-  };
-
-  const handleBlur = () => {
-    setKey(`${choice.value}:${choice.synonyms ? choice.synonyms.join() : ''}`);
-    if (!choice.value && (!choice.synonyms || !choice.synonyms.length)) {
-      onDelete(index);
-    }
-  };
+    [choiceType, onchange, setChoiceType]
+  );
 
   return (
-    <div css={[choiceItemContainer(), choiceItem]} key={key}>
-      <div css={choiceItemValue}>
-        <EditableField
-          onChange={handleEdit('value')}
-          value={choice.value}
-          styles={{
-            root: { margin: '7px 0 7px 0' },
-          }}
-          onBlur={handleBlur}
+    <React.Fragment>
+      <div style={{ display: 'flex', justifyContent: 'space-between', padding: '5px 0' }}>
+        <WidgetLabel
+          label={label}
+          description={formatMessage(
+            "A list of options to present to the user. Synonyms can be used to allow for variation in a user's response."
+          )}
```

Comment:
This may need to correlate with the choice type because this doesn't make sense with an expression.